midi_app_controller/state/state_manager.py

```diff
@@ -242,14 +242,22 @@ def load_state(self):
         # want to use them. So we make sure that all file paths are ok for us
         # to use.
         controller_file = state.selected_controller_path
-        if not any(is_subpath(d, controller_file) for d in Config.CONTROLLER_DIRS):
+        if controller_file is None:
+            return
+
+        if not any(
+            d and is_subpath(d, controller_file) for d in Config.CONTROLLER_DIRS
```

Comment:
We don't expect to store `None` in `Config.CONTROLLER_DIRS`. The `d and` is really not needed. I would rather get an exception if this happens.